src/controllers/post/getExplorePosts.ts

```diff
@@ -0,0 +1,22 @@
+import { type Request, type Response, type NextFunction } from 'express'
+import { getExplorePostsQuery } from '../../queries/post'
+import CustomError from '../../helpers/CustomError'
+
+const getExplorePosts = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const { page } = req.query
+    const pageNumber = Number(page)
+    if (pageNumber < 0 || Number.isNaN(pageNumber)) {
+      throw new CustomError(401, 'Bad Request.')
+    }
+    const posts = await getExplorePostsQuery(pageNumber, 10)
```

Comment:
maybe the limit could be dynamic also by using a query string with a default value = 10 and max = 100 maybe